pkg/sentry/ca/certificate_authority.go

```diff
@@ -137,13 +146,36 @@ func shouldCreateCerts(conf config.SentryConfig) bool {
 	return len(b) == 0
 }
 
+func detectCertificates(path string) error {
+	t := time.NewTicker(time.Second * 1)
```

Comment:
Can you please define `time.Second * 1` as const?

Reply:
Done